src/api/nodeRpcCall.js

```diff
@@ -1334,6 +1334,48 @@ const applyForCongress = async (walletAddress, callback) => {
   });
 }
 
+const createLlmSpendingMotion = async ({ walletAddress, transferToAddress, transferAmount, spending }, callback) => {
+  const api = await getApi();
+  const injector = await web3FromAddress(walletAddress);
+
+  
+
+  const councilMembers = await api.query.council.members()
+  const threshold = Math.ceil(councilMembers.length / 2)
+
+  const executeProposal =
+  spending === "sendToAddress"
+    ? api.tx.llm.sendLlm(transferToAddress, transferAmount)
+    : spending === "sendToPolitipool"
+    ? api.tx.llm.sendLlmToPolitipool(transferToAddress, transferAmount)
+    : undefined;
+
+  if (!executeProposal) {
+    const details = "Incorrectly filled Liquidity"
```

Comment:
Let's just split it into two separate functions. Then this error won't be possible.

Something like:
```
const congressSendLlm = async ({ walletAddress, transferToAddress, transferAmount }, callback) => {
  const api = [...]
  const executeProposal = api.tx.llm.sendLlm(transferToAddress, transferAmount);
  [...]
}

const congressSendLlmToPolitipool = async ({ walletAddress, transferToAddress, transferAmount }, callback) => {
  const api = [...]
  const executeProposal = api.tx.llm.sendLlmToPolitipool(transferToAddress, transferAmount);
  [...]
}
```

I know it's a lot more code, but the correct way to avoid this duplication would be through wrappers and/or passing the executeProposal itself. Doing it with this `spending === "sendToAddress"` is very error-prone and can't be type checked.

Reply:
These should be triggered by 2 different redux actions. Then the component for the modal would have the `if` logic, making it all properly contained.